carbon-design-system/carbon-website
# fix(icons): update file path for glyphs

Closes https://github.com/carbon-design-system/carbon-website/issues/2847

#### Changelog

**Changed**

- update file path for glyph assets

--- 
To test, go to the icons page and:
1. search for "status"
1. scroll past the "senses" category
  
For both these cases, the page should load appropriately and not error in the console as described in the original issue report
